CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.12
+  - Ensure java 8 compabatibility [#143](https://github.com/logstash-plugins/logstash-filter-date/pull/143)
```

Comment:
```suggestion
  - Ensure java 8 compatibility [#143](https://github.com/logstash-plugins/logstash-filter-date/pull/143)
```

Reply:
That's some quality spelling, right there...